typescript/infra/src/agents/gcp.ts

```diff
@@ -101,6 +106,17 @@ export class AgentGCPKey extends CloudAgentKey {
         return Keypair.fromSeed(
           Buffer.from(strip0x(this.privateKey), 'hex'),
         ).publicKey.toBase58();
+      case ProtocolType.CosmWasm:
```

Comment:
This doesn't match the Protocol Type Yorke and I added on our branch. We called it `Cosmos`, not `CosmWasm`

Reply:
https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/2859/files

Reply:
I'll rename - fwiw this is just the enum variant name and the value is "cosmos" for consistency with what the agents want. But regardless I feel like calling cosmwasm "cosmos" is weird because of our work on cosmos sdk, which I guess we can just call "cosmossdk" when we ship that

Reply:
Hmm sorry I don't quite follow. Do you prefer `Cosmos = 'cosmos'`, `CosmWasm = 'cosmos', or `CosmWasm = 'cosmwasm'` here? I prefer the 1st one because the there are utils that will rely on this protocol type which are not specific to cosmwasm

Reply:
CosmWasm = 'cosmwasm' would've been my preference but I'll just stick with Cosmos = 'cosmos'